2025-04-01 15:46:04 [error]: Error: Unknown database 'my_database'
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:46:58 [error]: Error: Unknown database 'my_database'
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:49:17 [error]: Error: Unknown database 'my_database'
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:51:46 [[31merror[39m]: Error: Access denied for user 'root'@'localhost' (using password: YES)
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:52:14 [[31merror[39m]: Error: Access denied for user 'root'@'localhost' (using password: YES)
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:52:34 [[31merror[39m]: Error: Access denied for user 'root'@'localhost' (using password: YES)
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:53:11 [[31merror[39m]: Error: Access denied for user 'root'@'localhost' (using password: YES)
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:53:34 [[31merror[39m]: Error: Access denied for user 'root'@'localhost' (using password: YES)
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:54:52 [[31merror[39m]: Error: Access denied for user 'root'@'localhost' (using password: YES)
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:58:06 [[31merror[39m]: Error: Access denied for user 'root'@'localhost' (using password: YES)
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
2025-04-01 15:59:10 [[31merror[39m]: Error: Access denied for user 'root'@'localhost' (using password: YES)
    at Packet.asError (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packets\packet.js:740:17)
    at ClientHandshake.execute (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\commands\command.js:29:26)
    at PoolConnection.handlePacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:475:34)
    at PacketParser.onPacket (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:93:12)
    at PacketParser.executeStart (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\packet_parser.js:75:16)
    at Socket.<anonymous> (D:\Proyectos\business-management-system\backend\node_modules\mysql2\lib\base\connection.js:100:25)
    at Socket.emit (node:events:518:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
